jazz_deployments-event-handler/index.js

```diff
@@ -86,35 +83,35 @@ var processEventRecords = (event, configData, authToken) => {
 	});
 };
 
-var processEventRecord = (record, configData, authToken) => {
+function processEventRecord (record, configData, authToken){
```

Comment:
probably for another PR(refactor) but these event related functions can be grouped in a separate module

Reply:
WILL TAKE IT UP AS PART OF THE NEW STORY INTENDED TO REDUCE THE EXPORTED MODULES